src/scene.rs

```diff
@@ -166,6 +188,15 @@ impl<I: ImageSize> Scene<I> {
         total
     }
 
+    /// Get the number of running animations for a sprite. If sprite does not
+    /// exist, returns 0.
```

Comment:
either way - matching or unwrapping, it's still extra code you need to write.

But will change to your suggestion, I'm new to rust so don't have a good feel for what a "normal" API feels like yet.